interview/src/main/java/ua/edu/ukma/interview/InterviewApplication.java

```diff
@@ -7,6 +7,7 @@
 public class InterviewApplication {
 
     public static void main(String[] args) {
+          System.out.println("Hello World");
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 10, expected level should be 8.
<!-- __reviewdog__:ChA3YmFkZTY2OTFlNjllZDgxEgpjaGVja3N0eWxl -->